dll/win32/CMakeLists.txt

```diff
@@ -181,6 +181,7 @@ add_subdirectory(schannel)
 add_subdirectory(scrrun)
 add_subdirectory(secur32)
 add_subdirectory(security)
+add_subdirectory(sendmail)
```

Comment:
I'm not a fan of continuing putting any dlls into that directory dump called "dll/win32" and instead, I would prefer if we had a real directory called "shell" where we put all the stuff related to the shell (i.e. explorer, the shell extensions, that new dll, etc.)

Reply:
Create a JIRA issue for it, please.

Reply:
We have `dll/shellext/`.

Reply:
> Create a JIRA issue for it, please.

https://jira.reactos.org/browse/CORE-9111